clang/docs/UsersManual.rst

```diff
@@ -1157,19 +1156,16 @@ existed.
   #if foo
   #endif foo // warning: extra tokens at end of #endif directive
 
-  #pragma clang diagnostic push
-  #pragma clang diagnostic ignored "-Wextra-tokens"
+  #pragma GCC diagnostic push
+  #pragma GCC diagnostic ignored "-Wextra-tokens"
 
   #if foo
   #endif foo // no warning
 
-  #pragma clang diagnostic pop
+  #pragma GCC diagnostic pop
```

Comment:
I think we want to mention that `#pragma clang diagnostic` also exists so that users know what it is when they come across it and how it differs from `#pragma GCC diagnostic` in terms of extra functionality. WDYT?

Reply:
Makes sense, even if we say elsewhere that `GCC` and `clang` can be used interchangeably for pragmas supported by both compilers.

Reply:
Done